src/mapcraftercore/renderer/blocktextures.cpp

```diff
@@ -197,6 +229,22 @@ BlockTextures::BlockTextures()
 	  GLASS_SILVER("glass_silver"),
 	  GLASS_WHITE("glass_white"),
 	  GLASS_YELLOW("glass_yellow"),
+      GLAZED_TERRACOTTA_BLACK("glazed_terracotta_black"),
```

Comment:
I actually did not know, that explains the indentation errors here. Normally my IDE should detect these automagically, but apparently that failed. I did not knew that `make gen_texture_code` would update the source files, I will update this.